clearpath_platform/launch/platform.launch.py

```diff
@@ -0,0 +1,80 @@
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument, ExecuteProcess, GroupAction, IncludeLaunchDescription, LogInfo
+from launch.conditions import LaunchConfigurationEquals
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch.substitutions import EnvironmentVariable, FindExecutable, LaunchConfiguration, PathJoinSubstitution
+from launch_ros.actions import Node
+from launch_ros.substitutions import FindPackageShare
+
+def generate_launch_description():
+    robot_model = LaunchConfiguration('robot_model')
+
+    # Launch Arguments
+    arg_robot_model = DeclareLaunchArgument(
+        'robot_model',
+        choices=['a200', 'j100'],
+        default_value='a200'
+    )
+    robot_model = LaunchConfiguration('robot_model')
+ 
+    log_platform_model = LogInfo(msg=["Launching Clearpath platform model: ", LaunchConfiguration('robot_model')])
+
+    # Launch clearpath_control/control.launch.py which is just robot_localization.
+    launch_clearpath_control = IncludeLaunchDescription(
+        PythonLaunchDescriptionSource(PathJoinSubstitution(
+        [FindPackageShare("clearpath_control"), 'launch', 'control.launch.py'])))
+
+    # Launch clearpath_control/teleop_base.launch.py which is various ways to tele-op
+    # the robot but does not include the joystick. Also, has a twist mux.
+    launch_clearpath_teleop_base = IncludeLaunchDescription(
+        PythonLaunchDescriptionSource(PathJoinSubstitution(
+        [FindPackageShare("clearpath_control"), 'launch', 'teleop_base.launch.py'])))
+
+    # Launch clearpath_control/teleop_joy.launch.py which is tele-operation using a physical joystick.
+    launch_clearpath_teleop_joy = IncludeLaunchDescription(
+        PythonLaunchDescriptionSource(PathJoinSubstitution(
+        [FindPackageShare("clearpath_control"), 'launch', 'teleop_joy.launch.py'])))
+
+    # Group for actions needed for the Clearpath Jackal J100 platform.
+    group_j100_action = GroupAction(
```

Comment:
I think we should have just a `group_platform_action` with all nodes in it. Then for nodes such as `micro_ros_agent` you can add a condition for  robot_model==j100